### ISSUE TITLE
Crash when allowing Storage Permission

### ISSUE THREAD

**Issue by Rahix** [siacs/Conversations#3350]
After updating to version *2.3.11*, I was no longer able to use Conversations without it crashing a few seconds after starting.  Turning off the `Storage` permission "fixes" the issue temporarily.  As soon as I turn it back on, Conversations starts to crash again.  Cleaning the `Conversations/` folder also did not help.

#### General information
* **Version:** 2.3.11
* **Device:** Google Nexus 5
* **Android Version:** Lineage 14.1-20181226
* **Server name:** self hosted (not relevant)
* **Server software:** prosody (not relevant)
* **Installed server modules:** -/- (not relevant)
* **Conversations source:** F-Droid

#### Steps to reproduce
1. (Not necessary, crash happens either way): Delete `Conversations/` folder.
2. Allow Conversations to access `Storage`
3. Start Conversations.

#### Expected result
Conversations not crashing.

#### Actual result
Conversations crashes after a few seconds with a `NullPointerException`.

#### Debug output
```
01-21 20:12:14.968  7562  7600 E AndroidRuntime: Process: eu.siacs.conversations, PID: 7562
01-21 20:12:14.968  7562  7600 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.startsWith(java.lang.String)' on a null object reference
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.persistance.FileBackend.getFileForPath(FileBackend.java:482)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.persistance.FileBackend.getFileForPath(FileBackend.java:477)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.services.XmppConnectionService.checkForDeletedFiles(XmppConnectionService.java:1072)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.services.XmppConnectionService.lambda$o-uIuPrJzYqCh3TTkc9haNUmZGU(XmppConnectionService.java)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.services.-$$Lambda$XmppConnectionService$o-uIuPrJzYqCh3TTkc9haNUmZGU.run(lambda)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at eu.siacs.conversations.utils.SerialSingleThreadExecutor$Runner.run(SerialSingleThreadExecutor.java:58)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
01-21 20:12:14.968  7562  7600 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:761)
01-21 20:12:14.970   784 19326 W ActivityManager:   Force finishing activity eu.siacs.conversations/.ui.ConversationsActivity
```

If there is any more info I can provide, I will happily do so!

**Comment by licaon-kter** [comment]
/duplicate #3345

Fixed in 2.3.12

**Comment by Rahix** [comment]
Oh, sorry, did not see that because it was already closed ...
